### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by markmandel**
Updates for the 0.2.0 release 🥳

Work on #398

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,91 @@
...
 
-## [v0.1.0](https://github.com/googleforgames/quilkin/tree/v0.1.0) (2021-07-08)
+## [v0.2.0](https://github.com/googleforgames/quilkin/tree/v0.2.0) (2021-09-21)
+
+[Full Changelog](https://github.com/googleforgames/quilkin/compare/v0.1.0...v0.2.0)
+
+**Breaking changes:**
+
+- Move I/O and configuration out of runner::run [\#350](https://github.com/googleforgames/quilkin/pull/350) ([XAMPPRocky](https://github.com/XAMPPRocky))
+
```
Comment (iffyio) [inline CHANGELOG.md:10]:
```suggestion
- Changed the `FilterFactory::create_filter` function return type from `Result<Box dyn Filter>, Error> to `Result<FilterInstance, Error>` [\#396](https://github.com/googleforgames/quilkin/pull/396) ([iffyio](https://github.com/iffyio))

```

**Review by iffyio — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (iffyio) [reply CHANGELOG.md:10]: Aha I had no idea the label was a thing, I called the breaking change out explicitly in the the #396 PR

Comment (markmandel) [reply CHANGELOG.md:16]: Pushed up an updated changelog, PTAL - if you are happy, I'll finish the release 👍🏻

Comment (iffyio) [reply CHANGELOG.md:16]: LGTM! Thanks :)

**Review by iffyio — APPROVED** [review]

**Pull request merged**